Rakefile

```diff
@@ -50,17 +50,17 @@ namespace :crowbar do
     ).flatten.compact.uniq
   end
 
-  desc "Init all barclamps"
+  desc "Initialize all barclamps (:fork, :clone, :add_upstream, :add_susecloud)"
```

Comment:
No, rake -P is a standard command. Everybody who wnats to see details can see it directly at rake -h